src/component/registry/types.ts

```diff
@@ -378,6 +378,16 @@ export interface ShapeBpmnSemantic extends BaseBpmnSemantic {
   eventDefinitionKind?: ShapeBpmnEventDefinitionKind;
   incomingIds: string[];
   outgoingIds: string[];
+  /**
+   * This is the ID of the direct parent of the current element, which can be a:
+   * - process/participant
+   * - lane
+   * - sub-process or call activity
```

Comment:
**suggestion:** To be consistent with the other
```suggestion
   * - sub-process 
   * - call activity
```

Reply:
done in b8bb428e